### DISCUSSION TITLE
Error with Making free database

### DISCUSSION THREAD

**Discussion by PriyanshVats** [planetscale/discussion#605] [Q&A] [answered]
![image](https://github.com/planetscale/discussion/assets/158857620/009b7c3d-7f7f-4803-ba0a-43eaa138854b)

**Comment by jgreet** [comment, marked as answer]
Hello,

This error indicates that the card you're trying to add is already associated with another organization on PlanetScale. 
 
You can add the same card to multiple organizations, but only once you have successfully paid an invoice with it.
